.github/workflows/main-workflow.yaml

```diff
@@ -11,6 +11,8 @@ jobs:
   bump-dev-version: # only do that when actually merging in main/develop branch
     if: github.event_name != 'pull_request'
     uses: ./.github/workflows/bump-dev-version.yaml
+    secrets: inherit
```

Comment:
This inherits the secrets from the organization?

Reply:
Yes

Reply:
_Environment variables_ from the organization are inherited automatically?

Reply:
Yes:
> To reuse variables in multiple workflows, set them at the organization, repository, or environment levels and reference them using the vars context. For more information see "[Variables](https://docs.github.com/en/actions/learn-github-actions/variables)" and "[Contexts](https://docs.github.com/en/actions/learn-github-actions/contexts#vars-context)."

from [here](https://docs.github.com/en/actions/using-workflows/reusing-workflows#limitations)